src/rdkafka_int.h

```diff
@@ -339,6 +339,38 @@ struct rd_kafka_s {
                 /**< Lock for sparse_connect_random */
                 mtx_t         sparse_connect_lock;
         } rk_suppress;
+
+        /**
+         * SASL/OAUTHBEARER token value, token metadata, and extensions
+         */
+        struct {
+                /**< The b64token value as defined in RFC 6750 Section 2.1
+                 *   https://tools.ietf.org/html/rfc6750#section-2.1
+                 */
+                char *token_value;
+                /**< When the token expires, in terms of the number of
+                 *   milliseconds since the epoch.
```

Comment:
Added this:

"See https://currentmillis.com/."

Unless I'm misunderstanding (which is possible), "number of milliseconds since the epoch" is by definition both wallclock and independent of timezone.  For example, if a remote server in US West determines the token expiration should be 2019-02-11T**12**:30:00.00 in its timezone, that will correspond to `md_lifetime_ms=1549906200000`, and a corresponding JWT would have the claim `"exp":1549906200` (it is the number of seconds rather than milliseconds -- potentially fractional, but not in this case).  Then, if the client and the broker are running in US East, which is 3 hours later timezone-wise, the client and broker will see the expiration time as 2019-02-11T**15**:30:00.00 in its timezone (if it were to convert it to a local time, which it probably would not do so except to maybe log it for debugging or informational purposes -- it would instead do all calculations in milliseconds-since-epoch).

Let me know if I missed/misunderstood something.